project_milestone/models/project_task.py

```diff
@@ -0,0 +1,22 @@
+from odoo import models, fields, api
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    milestone_id = fields.Many2one('project.milestone',
+                                   string="Milestones",
+                                   group_expand='_read_group_milestone_ids',
+                                   domain="[('project_id', '=', project_id)]")
+    use_milestones = fields.Boolean(string="Use Milestones",
+                                    related='project_id.use_milestones',
+                                    help="If enabled, then Milestones will be \
+                                          available. Related to the Project \
+                                          'Use Milestones' field.")

```

Comment:
Same here

Reply:
Same here, when I open this in editors, there is no additional characters after ')' that i can delete. Ideas?